src/components/battles/animations/shared/Fainting.ts

```diff
@@ -57,4 +67,31 @@ export class Fainting<TGameStartr extends FullScreenPokemon> extends Component<T
 
         this.gameStarter.modAttacher.fireEvent(this.gameStarter.mods.eventNames.onFaint, pokemon, battleInfo.teams.player.actors);
     }
-}
+
+    /**
+     * Sees if battle is against trainer or wild Pokemon.
+     *
+     * @param teamName    Tells if the Pokemon is the players or the opponents.
+     * @param battleInfo    Used to tell if player is fighting a wild Pokemon or trainer.
+     * @param pokemon   Information of defeated Pokemon.
+     */
+    private isThereATrainer(teamName: string, battleInfo: ITeamLeader | undefined , pokemon: IPokemon): IExperienceGains[] {
+        if (teamName === "opponent") {
+            let trainer: boolean = true;
+            if (battleInfo === undefined) {
+                trainer = false;
+            }
+            return this.gameStarter.experience.calculateExperience(pokemon, trainer);
+        }
+
+        return [];
+    }
+
+    /**
+     * Changes current battle participant list to selected Pokemon.
+     */
+    private OpponentFainted(): void {
+        this.gameStarter.itemsHolder.setItem("BattleParticipants",
+                                             this.gameStarter.battleMover.getBattleInfo().teams.player.selectedActor);
```

Comment:
Alignment is off here because you're the first argument, in this case `"BattleParticipants"`, is what it's all checked against. Move it to the next line.